### PR TITLE
scripted-diff: Rename key size consts to be relative to their class

### PR DESCRIPTION
**Pull request by Empact**
~~And introduce CPubKeySig to host code relative to key sigs.~~

### CODE REVIEW

File: src/pubkey.h [hunk 1]
```diff
@@ -26,40 +26,48 @@ class CKeyID : public uint160
 
 typedef uint256 ChainCode;
 
+class CPubKeySig
```
Comment (jonasschnelli) [inline src/pubkey.h]: Why not a namespace?

**Comment by jonasschnelli** [comment]
Concept ACK.
Not sure if `CPubKeySig` needs its own space.

**Comment by Empact** [comment]
Thanks, made PubKeySig a namespace.

**Comment by Empact** [comment]
On second thought, dropped PubKeySig - without any other code it's barely a separate concern.

**Comment by instagibbs** [comment]
seemingly unrelated travis error:
>File "/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/rpc_net.py", line 54, in _test_getnettotals
>    assert_equal(before['bytesrecv_per_msg']['pong'] + 32, after['bytesrecv_per_msg']['pong'])

kicked travis

**Comment by Empact** [comment]
@instagibbs thanks - seems like that is a timing-related failure, I pushed #12545 to increase the wait, and reduce the likelihood of spurious failures.

**Comment by MarcoFalke** [comment]
Needs rebase if still relevant

**Comment by Empact** [comment]
Rebased

**Comment by Empact** [comment]
Rebased & reimplemented as a scripted-diff

**Comment by sipa** [comment]
utACK f1583f26f2d9cb199ace6054a67797e1d4d9bed0

**Comment by Empact** [comment]
Rebased

**Comment by Empact** [comment]
Rebased to confirm scripted-diff still applies. It does. 👍

**Comment by MarcoFalke** [comment]
utACK 2dda5dd034, but please note that the scripted diff will modify files outside the git tree. I'd prefer to avoid that.

**Comment by Empact** [comment]
Updated to include 2 new cases, and brought it down to a single `sed` call which only acts on files in the repo.

**Comment by Empact** [comment]
Rebased for #14195

File: src/pubkey.h [hunk 2]
```diff
@@ -33,33 +33,33 @@ class CPubKey
     /**
      * secp256k1:
      */
-    static constexpr unsigned int PUBLIC_KEY_SIZE             = 65;
-    static constexpr unsigned int COMPRESSED_PUBLIC_KEY_SIZE  = 33;
+    static constexpr unsigned int SIZE             = 65;
+    static constexpr unsigned int COMPRESSED_SIZE  = 33;
```
Comment (ken2812221) [inline src/pubkey.h]: nit: Could have another commit to fix the format here

**Review by ken2812221 — APPROVED** [review]
utACK 40c69642bd4bed49d17b75e3f544afbc8d10e8ae

**Comment by Empact** [comment]
Rebased, added a commit to fixup whitespace.

**Comment by meshcollider** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/12461/commits/0580f86bb48004b797d4cb6273e7ffee0b0a0584

**Pull request merged**